### PR TITLE
Bug 1552888 - Typography and padding adjustments

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Pending design review.

File: content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss [hunk 1]
```diff
@@ -86,7 +86,7 @@ $excerpt-line-height: 20;
     display: flex;
     flex-direction: column;
     flex-grow: 1;
-    padding: 12px;
+    padding: 16px;
```
Comment (wolasi) [inline content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss]: Ticket specified making the right and left padding 16px;

**Comment by gvn** [comment]
Whoever has time to review...thx!

File: content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss [hunk 2]
```diff
@@ -98,6 +98,10 @@ $excerpt-line-height: 20;
     .excerpt {
       // show only 3 lines of copy
       @include limit-visibile-lines(3, $excerpt-line-height, $excerpt-font-size);
+
+      @at-root .ds-card-grid-divisible-by-3 & {
```
Comment (punamdahiya) [inline content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss]: may be add a line here what this does or what it compiles to

Comment (gvn) [reply content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss]: `@at-root` is part of Sass, but I can document it if feels necessary :)

**Comment by Mardak** [comment]
Why not put the card grid styling next to the other card grid sizing for title?
https://github.com/mozilla/activity-stream/blob/998ac8414030a808f4c97eef0b5070266f768f8e/content-src/components/DiscoveryStreamComponents/CardGrid/_CardGrid.scss#L71-L72

**Comment by gvn** [comment]
Sure, that makes sense as well.

**Pull request merged**